test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -19,6 +19,832 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
     {
+        public static TheoryData OptOutPartialTagDataCSharp
```

Comment:
oh, and `"@{<!p /}"` as well as the analogous in HTML (no missing end tag cases there of course)